### ISSUE TITLE
Rails 6.0 compatibility

### ISSUE THREAD

**Issue by d4be4st** [infinum/enumerations#43]
Got an error:

DEPRECATION WARNING: Delegating missing fetch_foreign_key_values method to Model. Accessibility of private/protected class methods in :scope is deprecated and will be removed in Rails 6.0. 

For this line of code:
https://github.com/infinum/enumerations/blob/master/lib/enumerations.rb#L154

Basically we need to make fetch_foreign_key_values a private method
